invenio_app_rdm/requests_ui/views/requests.py

```diff
@@ -49,7 +49,7 @@ def _resolve_topic_record(request):
     user_owns_request = str(creator_id) == str(current_user.id)
 
     if request["is_closed"] and not user_owns_request:
-        return dict(permissions={}, record_ui=None)
+        return dict(permissions={}, record_ui=None, record=None)
```

Comment:
Fixing small overlook from this PR: https://github.com/inveniosoftware/invenio-app-rdm/commit/05b4d50c8f50f3571255b5e68f967f721f0a4f0b#diff-4a94629a771abc41b540cb59d363d886634610dd06924aad8305ccae9250ced7L117